modules/core/src/main/java/org/apache/ignite/internal/processors/cache/GridCachePartitionExchangeManager.java

```diff
@@ -3484,6 +3462,31 @@ else if (r != null) {
                 }
             }
         }
+
+        /**
+         * Rebalance is not required on client node and always required when exchange future is null.
+         * In other cases the method checks all caches and decides required rebalance or not for specific exchange.
```

Comment:
I suggest to correct this description as follows:

Rebalancing is not required on the client node and is always required when the exchange future is null. In other cases,
this method checks all caches and decides whether rebalancing is required or not for a specific exchange.